sigar_linux_common_test.go

```diff
@@ -0,0 +1,101 @@
+package gosigar_test
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	sigar "github.com/elastic/gosigar"
+	"github.com/stretchr/testify/assert"
+)
+
+var mtabDir string
+var oldMtabf string
+
+func setUpCommonTest(t testing.TB) {
+	var err error
+	oldMtabf = sigar.Mtabf
+	mtabDir, err = ioutil.TempDir("", "sigarTests")
+	sigar.Mtabf = mtabDir + "/mtab"
```

Comment:
Nit. Not really neccesary, but just in case, you could add a cleanup function to revert to the old value.
```suggestion
	sigar.Mtabf = mtabDir + "/mtab"
	t.Cleanup(func() { sigar.Mtabf = oldMtabf })
```